ChangeLog

```diff
@@ -5,6 +12,26 @@
     modifications to Hashtable that might exposure potential memory corruption 
     with default copy constructor
 
+2015-01-08  Daniel Standage <daniel.standage@gmail.com>
+
+    * khmer/file.py -> khmer/kfile.py: renamed internal file handling
+    class to avoid collisions with builtin Python file module
+    * sandbox/collect-reads.py, sanbox/saturate-by-median.py,
+    sandbox/sweep-files.py, sandbox/sweep-reads.py,
+    scripts/abundance-dist-single.py, scripts/abundance-dist.py,
+    scripts/annotate-partitions.py, scripts/count-median.py,
+    scripts/count-overlap.py, scripts/do-partition.py,
+    scripts/extract-long-sequences.py, scripts/extract-paired-reads.py,
+    scripts/extract-partitions.py, scripts/filter-abund-single.py,
+    scripts/filter-abund.py, scripts/filter-stoptags.py,
+    scripts/find-knots.py, scripts/interleave-reads.py,
+    scripts/load-graph.py, scripts/load-into-counting.py,
+    scripts/make-initial-stoptags.py, scripts/merge-partitions.py,
+    scripts/normalize-by-median.py, scripts/partition-graph.py,
+    scripts/sample-reads-randomly.py, scripts/split-paired-reads.py,
+    tests/test_script_arguments.py, tests/test_scripts.py: changed all
```

Comment:
For future reference: this list can be collapsed a bit by use of BASH glob syntax (see below). No need to change it now.